src/components/Dashboard/Sidebar.tsx

```diff
@@ -0,0 +1,164 @@
+import { NavLink } from 'react-router-dom';
+import React, { useState } from 'react';
+import { motion } from 'framer-motion';
+import {
+  FaSignOutAlt, FaTable, FaOutdent, FaTh, FaRegSun,
+  FaRocketchat, FaBullhorn, FaRegUser, FaMoon, FaShoppingCart, FaHouseUser,
+} from 'react-icons/fa';
+import { MdOutlineApps } from 'react-icons/md';
+import SubMenu from './Dropdownmen';
+import { ThemeToggle } from '../navbar/ThemeToggle';
+
+function Sidebar() {
+  const sidebarAnimation = {
+    open: {
+      width: '15rem',
+      transition: {
+        damping: 40,
+      },
+    },
+    closed: {
+      display: 'none',
+      transition: {
+        damping: 40,
+      },
+    },
+  };
+
+  const [isOpen, setOpen] = useState(false);
+
+  const subMenuList = [
+    {
+      name: 'Product',
+      icon: FaTable,
+      menus: ['Add Product', 'Product List'],
+    },
+    {
+      name: 'Orders',
+      icon: FaShoppingCart,
+      menus: ['All Orders', 'Concered Orders'],
+    },
+    {
+      name: 'Users',
+      icon: FaHouseUser,
+      menus: ['All users', 'Buyer', 'Seller', 'Dissabled user'],
+    },
+  ];
+  const datadown = [{
+    icon: FaRegSun,
+    name: 'Settings',
+  },
+  {
+    icon: FaRegUser,
+    name: 'Account',
+  },
+  {
+    icon: FaSignOutAlt,
+    name: 'Logout',
+  },
+  {
+    icon: FaMoon,
+    name: 'switch mode',
+  },
+
+  ];
+  return (
+
+    <div>
+      {!isOpen && (
+        <motion.div
+          onClick={() => setOpen(true)}
+          className="absolute w-fit h-fit z-50 left-0 top-19 cursor-pointer "
+        >
+          { /* for close and open botton */}
+          <MdOutlineApps size={30} className="fill-cyan-500" />
+        </motion.div>
+      )}
+
+      <motion.div
+        variants={sidebarAnimation}
+        animate={isOpen ? 'open' : 'closed'}
+        className="flex flex-low h-[100vh] px-5 py-2 overflow-hidden md:relative fixed"
```

Comment:
I don't know if you noticed, but this menu is not actually fixed.

You have to scroll the page in order to see the entire menu.

And speaking of which, when you scroll, the navbar's z-index conflicts with the sidebar's z-index, with the menu partly going underneath, but the icon remain visible over everything.

Find a way to give the menu a proper fixed position, and make sure the content within fits.